### DISCUSSION TITLE
bewly浏览器图形加速卡顿问题

### DISCUSSION THREAD

**Discussion by MisakaSakura** [BewlyBewly/BewlyBewly#1173] [Q&A] [answered]
你们直接关闭了我后续回复的权限有点离谱的。

首先我承认上个issue我确实在方向上出错了，我也是后面才意识到的。

先摆出一些基础事实：我是thinkbook 16P（i7-14650hx+4060，32G），电脑打开独显直连，游戏默认全屏窗口化。

关闭浏览器图形加速原因：双屏时，主屏进行GPU高占用的活动，也就是游戏时，同时副屏播放网页视频或者直播，会导致副屏的视频画面卡顿无法正常观看，这实际上是Windows或者chromium内核浏览器的bug（也可能跟两个屏幕的刷新率不一致有关），目前网上能找到的唯一可用解决办法只有关闭浏览器的图形加速功能。

跟bewly有关的：bewly的模糊特性依赖浏览器的图形加速功能，因为我一段时间前关闭了体现加速，而我不是很经常使用电脑访问B站所以最近才发现bewly的导航栏功能模块非常卡顿，排查之后发现是跟图形加速关闭有关系。

最终结论：目前就是我的使用场景图形加速跟副屏播放视频不可得兼，这跟是否关闭bewly的模糊无关，因为
1.我的机器性能支持同时开启bewly的全部模糊特性+重度GPU活动，这是经过验证的
2.即使把bewly的模糊特性关掉也无法解决我副屏在主屏进行重度GPU工作时副屏播放视频的卡顿现象（视频不开全屏勉强能看）
3.只有关掉图形加速才能让双屏的工作都正常，代价就是失去图形加速支持的bewly无法保证模糊开启时不卡顿。

我一开始其实就是抱着找问题原因来的，第一时间锁定到问题所在后我其实不是很指望你们能解决了，因为这跟Windows或者chromium的功能冲突有关，可能后续说明上跟原问题有点偏差，给你们造成了一些误解和困扰，先在这里给积极要解决问题的项目成员表示感谢，这个讨论也只是为了给后续滑坡向错误方向的issue一个正本清源的机会。

**Comment by hakadao** [comment]
不知道你是否試過原本b站不使用插件+不使用圖形加速情況下與windows全螢幕遊戲是否也會導致卡頓
我承認我確實lock了這個issue會導致你以爲不想解決問題，但是圖形加速=使用gpu渲染，你如果玩現在遊戲不使用gpu用純cpu渲染能有多大改進空間？
GitHub我常常看不少issue確實現在都沒有多少耐性解答，首先確實多數人issue標題寫得模棱兩可或簡單說一兩個字，還有人issue模板標題也不改，還是「請在這裏輸入你的問題」類似字樣

人在沒有取得相對應報酬和金錢資助的話，不是十分有必要和商業產品做得那麼1對1客服那麼耐心溝通的。我就做了開源放了出來，各自沒有欠各自，何況看到模棱兩可問題標題和類似不用GPU使用Windows毛玻璃效果很卡能否解決或優化

**Comment by hafterain** [comment, marked as answer]
试试：

- **关闭独显直连**后，能正常工作吗；
- 另外，尝试一下[此文章](https://www.bilibili.com/read/cv11918725/)中的方法，为你的 Google Chrome 开启强制 GPU 渲染（先尝试指定你本机**核显**〔在关闭独显直连的情况下〕），看看能否正常工作。

**Reply by MisakaSakura** [reply]
> 试试：
> 
> - **关闭独显直连**后，能正常工作吗；
> - 另外，尝试一下[此文章](https://www.bilibili.com/read/cv11918725/)中的方法，为你的 Google Chrome 开启强制 GPU 渲染（先尝试指定你本机**核显**［在关闭独显直连的情况下］），看看能否正常工作。

似乎找到问题根源了，我开启了N卡的视频超帧功能，也就是播放视频时会有个「增强视频」的选项，单独游戏或者单独视频都不会有影响，但是同时启动游戏和播放视频开启这个功能时视频就会卡顿了，把这个功能关闭视频就正常了。

**Reply by hafterain** [reply]
請問能否截相關選項的圖以便我加入Wiki的常見問題頁面？

如果方便提供的話，非常感謝。

**Reply by MisakaSakura** [reply]
> 請問能否截相關選項的圖以便我加入Wiki的常見問題頁面？
> 
> 如果方便提供的話，非常感謝。

![image](https://github.com/user-attachments/assets/85595bbc-6f8d-40c7-ad59-73a6db95cc89)
![image](https://github.com/user-attachments/assets/c452d738-24c2-4aab-874f-c2be02f5963e)
![image](https://github.com/user-attachments/assets/40ac02e5-30db-4e40-aab9-0b1245a54981)

就这东西，好像浏览器设置里开关无所谓，视频播放的时候这个增强不打开就不会影响播放

**Reply by hafterain** [reply]
謝謝你。
